src/app/containers/IndexPageSection/index.jsx

```diff
@@ -191,7 +196,16 @@ const IndexPageSection = ({ bar, group, sectionNumber }) => {
   // If this group does not have a strapline; do not render!
   // This may change in the future, if a way to avoid breaking UX is found.
   // Also, don't render a section without any items.
-  if (!(strapline && items) || items.length === 0) {
+
+  // If renderWithoutStrapline is false (which it is by default),
+  // then the section will not render if there is no strapline.
+  // If renderWithoutStrapline is true, and there is no strapline present,
+  // the section will always render.
+  if (!strapline && !renderWithoutStrapline) {
```

Comment:
Is there any reason we can't engineer this so `strapline` is the only control? The [negatively named](http://davej.io/2018/05/negative-naming.html) boolean makes this challenging to read.

Reply:
It is likely this might change - we want to start a conversation to see if the UX is acceptable without a strapline, and also to see how screenreaders handle it. 

If we get the green light to display sections without a strapline, then this logic can be completely removed. It was just to allow us to maintain the existing behaviour for Front Pages & IDX Pages while still letting us get FIX pages onto test.